### ISSUE TITLE
Display initial image

### ISSUE THREAD

**Issue by GeekyMonkey** [ogix/ng2-fancy-image-uploader#15]
I want to use this on a form where the user may have already uploaded an image previously.  So they're not adding an image, but they can replace the image.  Is it possible with this control to display that initial image instead of the empty box with the "add" text?

**Comment by ogix** [comment]
If you are using reactive forms then it is possible to specify: formControlName="imageUrl".

**Comment by joaolori** [comment]
How can i implement this?

formControlName="imageUrl" ?

**Comment by joaolori** [comment]
Can u provide example for this @ogix

**Comment by enyachoke** [comment]
@GeekyMonkey  Did you figure this out

**Comment by enyachoke** [comment]
@GeekyMonkey  I got it 
```<fancy-image-uploader id="field_photoUrl" name="photoUrl"  [(ngModel)]="farmer.photoUrl" [options]="options" (onUpload)="onUpload($event)"></fancy-image-uploader>```

**Comment by GeekyMonkey** [comment]
No. Sorry. I gave up on this control and rolled my own. It wasn't a great fit for my use anyway.

**Comment by enyachoke** [comment]
Anyway if somebody needs it
```Javascript
 public fileControl = new FormControl();
 ngOnInit() {

        this.fileControl.setValue('url', { emitEvent: false});
      
    }
```
```html
<fancy-image-uploader id="field_photoUrl" name="photoUrl"  [formControl]="fileControl"  [options]="options" (onUpload)="onUpload($event)"></fancy-image-uploader>
```

**Comment by jirapatj** [comment]
i try from @enyachoke  suggestion. it does not working. I got some error. 
Can you help me to fix?

![screen shot 2018-06-23 at 00 07 02](https://user-images.githubusercontent.com/7393333/41789414-6c35d1d0-7679-11e8-9f24-ef9768e6d1dc.png)
![screen shot 2018-06-23 at 00 06 42](https://user-images.githubusercontent.com/7393333/41789415-6c6cc9ec-7679-11e8-8269-c506dc2122f9.png)
